### DISCUSSION TITLE
Donations

### DISCUSSION THREAD

**Discussion by jmgirard** [easystats/easystats#354] [General]
Hi all,

I am teaching a summer school on R that uses easystats heavily. I would like to support the project and am considering donating a % of my profits to the project (would likely be several thousand USD). Can you tell me a little more about how donations are allocated and spent?

**Comment by DominiqueMakowski** [comment]
Oh wow, well thanks for at least considering us! 

To be entirely honest we don't have any fund management system in place, mostly because we never had to deal with major funding.. (we haven't yet used the donations of our generous and kind sponsors 😉 ). As far as I remember, the only thing that we bought for easystast were some stickers and we paid with our money 😄 

But we need to change that as our project keeps growing. What would be good is to:
1. **Deal with donations/funds transparently**: setting something like https://opencollective.com/ so that it's all open amd can be managed as a team
2. **Clear and transparent roadmap on what we use/plan on using money for**: for instance we toyed with the idea of hiring someone to help with the development of more documentation / introductory materials (e.g., a book), or organizing a meeting/conference etc.

So thanks for asking the questions and forcing us to think and do something about that

@easystats/maintainers

**Reply by jmgirard** [reply]
Ok, the money would not come in until June so there is plenty of time to set things up.

**Reply by jmgirard** [reply]
To be clear, there would be no strings attached to my donation. But I would certainly feel a lot more comfortable with it if something like the OpenCollective were used. I'd also say that (separate from the topic of donations), I would be very interested in contributing to or even leading efforts on an open statistics textbook using easystats.

**Reply by rempsyc** [reply]
Open Collective is what for instance the _Psychological Science Accelerator_ (PSA) has been using: https://opencollective.com/psysciacc and it seems to be working quite well. Dom do you want to setup our account?

**Reply by DominiqueMakowski** [reply]
Will try now

**Reply by DominiqueMakowski** [reply]
We have this now: https://opencollective.com/easystats

**Reply by rempsyc** [reply]
We've got our first contributor already :D Now Dom you can test what to do with the money in the account!

**Comment by jmgirard** [comment]
Just checking in on this. I'll have the revenue arrive in late June/early July. I don't necessarily have to donate it immediately, but we should get the ball rolling.

**Reply by rempsyc** [reply]
> Can you tell me a little more about how donations are allocated and spent?

What would you say are your expectations in terms of donation allocations and spending? If we used the money to buy beer (tea) and snacks at an easystats get-together at some point in the future, how comfortable would you be with that? Or travel expenses for such a meetup? Also, do you have any other spending ideas that could inspire us? :)

**Reply by jmgirard** [reply]
I would be comfortable with any kind of "normal" expenses for a nonprofit organization, including but not limited to: paying for operating costs, setting up and providing hospitality (e.g., lodging, food, and drinks) for organization meetings, paying "salary" or "wages" to key organization personnel for work on organization-related projects, buying project hardware/software/services, creating relevant awards, creating "bounties" for desired project work to get done, funding a student or intern to work on the project, etc. My main concern would be making sure that the funds are fairly and transparently used (as defined by the _organization_, not by me).

**Reply by jmgirard** [reply]
If at all possible, my request would be to set it up so that donors can treat donations as charitable (for the tax implications).

**Reply by jmgirard** [reply]
"Your charitable giving will qualify for a tax deduction only if it goes to a tax-exempt organization, as defined by section 501(c)(3) of the Internal Revenue Code. Examples of qualified institutions include religious organizations, the Red Cross, nonprofit educational agencies, museums, volunteer fire companies and organizations that maintain public parks. An organization can be nonprofit without 501(c)(3) status, which can make it tricky to ensure your charity of choice counts."

**Reply by jmgirard** [reply]
This seems to imply that educational and scientific pursuits qualify as exempt purposes. 

https://www.irs.gov/charities-non-profits/charitable-organizations/exemption-requirements-501c3-organizations

"The exempt purposes set forth in section 501(c)(3) are charitable, religious, educational, scientific, literary, testing for public safety, fostering national or international amateur sports competition, and preventing cruelty to children or animals. The term charitable is used in its generally accepted legal sense and includes relief of the poor, the distressed, or the underprivileged; advancement of religion; advancement of education or science; erecting or maintaining public buildings, monuments, or works; lessening the burdens of government; lessening neighborhood tensions; eliminating prejudice and discrimination; defending human and civil rights secured by law; and combating community deterioration and juvenile delinquency."

**Reply by jmgirard** [reply]
However, it looks like the organization would need to either be based in the US or we would need to make a US-based "friends of" organization to accept donations from US donors and then transfer to a non-US organization. If you want the main organization based elsewhere, perhaps I could help set up the US-based "friends of" organization. 

https://www.venable.com/insights/publications/2019/07/charity-abroad-us-donors-options-for-international

**Reply by jmgirard** [reply]
"Contributions to Collectives hosted by OSC are not deductible as charitable contributions for U.S. federal income tax purposes."

https://opencollective.com/opensource

I wonder if there's a way around this, e.g., with a partner organization.

**Reply by jmgirard** [reply]
"You can accept grants as a fiscally-sponsored project with OSC as your host organization.  Before you apply for a grant, make sure they can give grant funding to organizations that are 501c6 (mutual-benefit nonprofit). Some grants can only be awarded to nonprofits that are 501c3 (public-benefit nonprofit). If you are a US-based project with an explicitly charitable mission, and you wish to qualify for charity grants and tax-deductible donations, you may wish you apply to Open Collective Foundation."

https://docs.oscollective.org/how-it-works/tax-info

@DominiqueMakowski Could we consider moving from OSC to OCF (linked below)?

https://opencollective.com/foundation

For more on the differences between 501c6 (OSC) and 501c3 (OCF), see:
https://nonprofitlawblog.com/comparing-501c3-vs-501c6-for-nonprofit-startups/

**Reply by DominiqueMakowski** [reply]
Mmh I'm not sure how to do that @rempsyc you're admin I think do you see a button to change?

**Reply by jmgirard** [reply]
This page seems relevant:
https://docs.opencollective.com/help/collectives/change-fiscal-host

**Reply by rempsyc** [reply]
Those are great ideas, thanks for the suggestions!

The Psychological Science Accelerator (PSA) had exactly this debate, about setting up as a charitable non-profit (501(c)3), or to use a fiscal sponsor, like the Open Collective to collect donations. After considering the pros and cons of each, they decided to opt for the fiscal sponsor, as it makes lots of things a lot simpler, including not having to submit yearly annual financial reports and so on, which can incur additional costs (of time and money) on the organization's side. I can share the detailed post about this if relevant. But I think that if the PSA chose not to go the charitable route, I think it will be even less likely that we do.

> Could we consider moving from OSC to OCF (linked below)?

Hum, I'll look further into what the differences are. I know one is the foundation and the other one is the open source platform/entity, but I didn't know you could be sponsored by the foundation instead of the collective.

**Reply by rempsyc** [reply]
It seems like it would be possible and that we would be eligible (curious why PSA didn't go this route though), but it comes with some limitations: more paperwork (probably), higher management fees, and must be US-based:

> We are only able to host US-based projects and fund US-based activity, not overseas expenses.

Dom wanna try applying anyway?

https://docs.opencollective.foundation/getting-started/how-to-apply

**Reply by jmgirard** [reply]
My understanding is that, through the Open Collective, you can use either OSC or OCF as the fiscal sponsor/host and they would do the same work for you. But OSC is itself a 501c6 whereas OCF is a 501c3. So, assuming you qualify for a 501c3 (which I believe we would, based on the stuff linked above), you can use OCF and the main difference would be allowing US donors to treat donations as charity.

**Reply by rempsyc** [reply]
Damn, the fact that I made a small test donation now makes it harder to switch to the new host because we have to empty the current balance first 🙈

**Reply by DominiqueMakowski** [reply]
can you cash it out to yourself?

**Reply by jmgirard** [reply]
It sounded like you could transfer the balance to the new host.

"The new host should file an expense for the collective to withdraw the funds into their holding account, the new host can then allocate the funds using add funds"

https://docs.opencollective.com/help/collectives/change-fiscal-host

**Reply by rempsyc** [reply]
> It sounded like you could transfer the balance to the new host.

I thought so too, but it's actually donating it to "your Fiscal Host" = old fiscal host. That's what I did anyway (as per the instructions in the link). Edit: well anyway maybe I got confused haha. In any case the balance is clear now.

I've also removed our current fiscal sponsor. So Dom now you can apply to the new host (or I can do it if you prefer just let me know). There's a bit of info they're asking.

**Reply by jmgirard** [reply]
Regarding the international vs. US thing:

![image](https://github.com/easystats/easystats/assets/6829033/11833ab8-a4e6-4796-a012-2d097d43bad4)

Are any of the potential administrators based in the US?

Or, barring that, maybe we could make an international network and set up a US Easystats Chapter (and maybe Canadian, UK, etc. chapters as needed).

**Reply by rempsyc** [reply]
Brenton is in the US no? He's our only one... unless perhaps @jmgirard you want to be our US rep if they think we're not sufficiently US-based? :)

**Reply by jmgirard** [reply]
I would be very happy to do that if you're comfortable with it. Just don't want to overstep since I'm mostly a user and not developer.

**Reply by rempsyc** [reply]
Alright, it's done! I applied to move from OSC to OCF. They say that application reviews can take about 2 weeks 🤞

**Reply by jmgirard** [reply]
Thanks. Sorry to create extra work, but the ability to write off the donations is a BIG deal.

**Reply by rempsyc** [reply]
No worry, the extra work is tiny compared to the benefit (to you and incentive for future donors).

Only thing I'm not sure is I think we might have to disable donations through GitHub if everything now will have to go through OCF... Or redirect in some way.

**Reply by jmgirard** [reply]
I can move my monthly donations from GitHub to OC.

**Reply by rempsyc** [reply]
Ok, but let's wait to get the OCF approval first.

**Reply by rempsyc** [reply]
@jmgirard just an update: the OCF are reviewing our application and reached out to me with more questions. I have answered their additional questions and we are now on standby for their decision.

**Reply by jmgirard** [reply]
@rempsyc Sounds good. I got half the money in already, so will be ready for the donation soon (but can easily wait for OCF to decide).

**Reply by rempsyc** [reply]
@jmgirard OCF declined our application :(

Thoughts on where to go next? We could fall back on OSC but no tax deductions...

**Reply by DominiqueMakowski** [reply]
It got declined without any justifications really :(

**Reply by jmgirard** [reply]
That's a bummer. I'm teaching the workshop tomorrow but will look into some options after that.

**Reply by jmgirard** [reply]
Ok, I have been researching this. Instead of treating it as a charitable donation, I may be able to treat it as a business expense. So I could send money to your OSC and write that amount off as an expense so that it does not contribute to my adjusted gross income (which is the goal).

> "The IRS says that you can deduct a payment to a charity as a business expense when the payment to the charity bears direct relationship to your business and you make the payment with the reasonable expectation of financial return to commensurate the amount paid.
> 
> For example, Fisher CPA Firm decides to advertise that 10% of the sales to new customers in the month of December will be donated to Toys For Tot's. We run an email and print advertising campaign heading into December highlighting the virtues of doing business with Fisher CPA Firm because we support Toys for Tots and other charities in our community. Since we ran the campaign with the reasonable expectation that we will earn more money than the cost of the ad, we can deduct the 10% donation as advertising expense.
> 
> Another example, Fisher CPA Firm is asked to sponsor a small business conference put on by the local chamber of commerce (a 501c3 charity). We agree and pay a sponsorship fee because we believe the exposure from our
position as a supporter of small business will land us new clients that will earn us more than the cost of the sponsorship fee."
> --- https://www.fishercpafirm.com/blog/charitable-contributions-as-a-business-expense

Given that I did use the fact that a portion of the proceeds would be donated to easystats during advertising, my situation is quite similar to the first example. It is also the case that, by investing in easystats, my workshops go more smoothly and therefore help build my reputation and attract more business.

**Reply by rempsyc** [reply]
Alright, I will open back our OSC so you can proceed. Note that we are a non-profit but not a charity I believe since OCF declined our application.

**Reply by bwiernik** [reply]
I am not a lawyer or accountant, but my understanding is that it would be reasonable to list these as business expenses on the businesses expenses portion of the tax form

**Reply by jmgirard** [reply]
The main hurdle to cross is this:

> To be deductible, a business expense must be both ordinary and necessary. An ordinary expense is one that is common and accepted in your industry. A necessary expense is one that is helpful and appropriate for your trade or business. An expense does not have to be indispensable to be considered necessary.
> --- https://www.irs.gov/publications/p535 (What can I deduct?)

I think the easystats software is helpful and appropriate for my business and thus can be considered necessary. It is common and accepted in my industry to buy software, but since this is open source software, it's a little more dicey. But if my financial contributions to the project help make it more stable and comprehensive, then I think that's close enough.

**Reply by jmgirard** [reply]
@rempsyc Let me know when the OSC is open again.

**Reply by rempsyc** [reply]
@jmgirard Thanks, I had to reapply for OSC even, so just submitted the application.

**Reply by rempsyc** [reply]
@jmgirard we just got the OSC approval this morning, it's open now! https://opencollective.com/easystats

**Reply by jmgirard** [reply]
Awesome! Just made my contribution. If my workshops keep filling up, this should be a recurring (annual) thing.

**Reply by rempsyc** [reply]
That's amazing, thanks so much for this large donation @jmgirard, on behalf of all the @easystats/core-team! 🙏

**Comment by strengejacke** [comment]
Sorry, I haven't followed this discussion... But first of all, a big thank you also from my side! How can/do we proceed? We should make some plans soon, I'd say.
